dxp/how_to_minimize_bleeding_themes.md

```diff
@@ -0,0 +1,360 @@
+# How to Minimize Bleeding Themes
+
+> "Currently upgrading from DXP 7.0 to DXP 7.2 and they have a leaking theme. What I mean is that some styling they apply to the site also has an effect within the control panel."
+
+This is a common problem due to the global nature of CSS and Liferay's complexity. This document will outline some methods to minimize bleeding CSS.
+
+## Working With Clay CSS
+
+Liferay Portal admin controls use Clay CSS components extensively and aren't namespaced so modifying Clay CSS components directly will cause unintended issues. The way we can work around this is by namespacing our CSS selectors to only target the components we want to change.
+
+We shouldn't style HTML elements directly. For example:
+
+```css
```

Comment:
For these things that are actually _counter_-examples...

![counter-example](https://user-images.githubusercontent.com/7074/94237768-a9685200-ff0f-11ea-8f21-33e93dedf016.jpg)

... could we flag them explicitly like this (or similar)?

---

**⚠️ Counterexample:**

```css
a {
    color: orange;
}

/* etc ... */
```